pkg/agent/client.go

```diff
@@ -62,7 +74,10 @@ func inClusterConfig() (*rest.Config, error) {
 
 	// Agent is not able to verify Controller's cert as it's generated in-memory with loopback address.
 	// Use Insecure for now.
-	tlsClientConfig := rest.TLSClientConfig{Insecure: true}
+	tlsClientConfig := rest.TLSClientConfig{
+		CAFile:     "/var/run/antrea/secrets/ca.crt",
```

Comment:
is it possible for this file not to exist yet (if the agent starts before controller)? is it an issue?